configure.ac

```diff
@@ -49,6 +49,33 @@ AS_IF([test "x$enable_service" != "xno"], [
 	AC_DEFINE([ENABLE_SERVICE], [0])
 ])
 
+AC_ARG_ENABLE([bundle],
+	AS_HELP_STRING([--disable-bundle], [Disable bundle command])
```

Comment:
Just call it `--disable-create Disable bundle creation and modification` and control all three subcomands together.

Reply:
OK.